tests/test_utils.py

```diff
@@ -27,3 +31,29 @@ def test_nb_nll_moments(mu, alpha):
     assert np.abs(diff) < 0.2 * deviation
     error_var = np.abs(sample.var() - var_th) / var_th
     assert error_var < 1 / np.sqrt(n_montecarlo)
+
+
+# Test data loading from outside the package (e.g. on RTF)
+@mock.patch("pathlib.Path.is_dir")
+def test_rtd_example_data_loading(mocked_function):
+    """
+    Test that load_example_data still works when run from a place where the ``datasets``
+    directory is not accessible, as is when the documentation is built on readthedocs.
+    """
+
+    # Mock the output of is_dir() as False to emulate not having access to the
+    # ``datasets`` directory
+    pathlib.Path.is_dir.return_value = False
+
+    # Try loading data. Ignore flake8 "variable is assigned to but never used" error.
+    counts_df = load_example_data(  # noqa: F841
```

Comment:
instead of ignoring (noqa) don't assign any value (unless you want to check it for correctness in type, shape or anything else?

Reply:
indeed!